src/core/ext/filters/http/server/http_server_filter.cc

```diff
@@ -65,6 +65,9 @@ struct call_data {
 
   grpc_closure recv_trailing_metadata_ready;
   grpc_closure* original_recv_trailing_metadata_ready;
+
```

Comment:
Please remove blank line -- let's put all recv_trailing_metadata fields in one section.